src/components/ModalHeader.tsx

```diff
@@ -29,23 +29,24 @@ function ModalHeaderRef({ children, onClose, ...props }: ModalHeaderProps, ref:
       >
         {children}
       </P>
-      {typeof onClose === 'function' && (
-        <Flex
-          align="center"
-          justify="center"
-          padding="xsmall"
-          margin={-12}
-          borderRadius="medium"
-          cursor="pointer"
-          _hover={{ backgroundColor: 'fill-two-hover' }}
-          onClick={onClose}
-        >
-          <CloseIcon
-            size={14}
-            color="text-light"
-          />
-        </Flex>
-      )}
+      {/* Commented out as I think it might be needed at some point. */}
+      {/* {typeof onClose === 'function' && ( */}
+      {/*  <Flex */}
+      {/*    align="center" */}
+      {/*    justify="center" */}
+      {/*    padding="xsmall" */}
+      {/*    margin={-12} */}
+      {/*    borderRadius="medium" */}
+      {/*    cursor="pointer" */}
+      {/*    _hover={{ backgroundColor: 'fill-two-hover' }} */}
+      {/*    onClick={onClose} */}
+      {/*  > */}
+      {/*     <CloseIcon */}
+      {/*      size={14} */}
+      {/*      color="text-light" */}
+      {/*     /> */}
+      {/*  </Flex> */}
+      {/* )} */}
```

Comment:
Are you certain we don't need the close button anymore?

Reply:
That's what the issue says 🙂. I wouldn't completely remove it though.